src/Catalyst.Modules.Server.Blazor/Components/SendTransactionForm.razor.cs

```diff
@@ -57,34 +48,7 @@ public class SendTransactionFormBase : ComponentBase
 
         public void HandleValidSubmit()
         {
-            // //TODO: Currently there is no method to actually set the transaction amount
-            // var transaction = new TransactionBroadcast {Timestamp = Timestamp.FromDateTime(DateTime.UtcNow)};
-            // transaction.PublicEntries.Add(new PublicEntry
-            // {
-            //     Base = new BaseEntry
-            //     {
-            //         SenderPublicKey = PeerSettings.PublicKey.KeyToByteString()
-            //     }
-            // });
-            // var signingContext = new SigningContext
-            // {
-            //     NetworkType = PeerSettings.NetworkType,
-            //     SignatureType = SignatureType.ProtocolPeer
-            // };
-            //
-            // var signature = KeySigner.Sign(transaction.ToByteArray(), signingContext);
-            // transaction.Signature = new Signature
-            // {
-            //     RawBytes = signature.SignatureBytes.ToByteString(),
-            //     SigningContext = signingContext
-            // };
-            // var status = TransactionReceivedEvent
-            //    .OnTransactionReceived(transaction.ToProtocolMessage(PeerSettings.PeerId,
-            //         CorrelationId.GenerateCorrelationId()));
-            // JsRuntime.InvokeAsync<object>("window.alert", $"Transaction Status: {status.ToString()}")
-            //    .ConfigureAwait(false);
-            // Model = new SendTransactionModel();
-            // StateHasChanged();
+            // TODO: Currently there is no method to actually set the transaction amount
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4630536)